.travis.yml

```diff
@@ -46,7 +46,7 @@ matrix:
     - name: cargo +stable build --no-default-features
       rust: nightly
       env:
-        - RUSTFLAGS="-Z allow-features="
+        - RUSTFLAGS="-Z allow-features=proc_macro_span"
```

Comment:
It may be preferable to include a link to #1564 here.

Reply:
Put a TODO with link, I'll reuse that issue to track removing this once `proc_macro2` allows.